src/components/NewWorkspaceModal.js

```diff
@@ -32,48 +30,41 @@ const constraints = {
   }
 }
 
-export default _.flow(
-  Utils.withCancellationSignal,
-  Utils.connectStore(authStore, 'authState')
-)(class NewWorkspaceModal extends Component {
-  static propTypes = {
-    cloneWorkspace: PropTypes.object,
-    onDismiss: PropTypes.func.isRequired,
-    customMessage: PropTypes.string,
-    title: PropTypes.string,
-    buttonText: PropTypes.string
-  }
+const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
+  cloneWorkspace, onSuccess, onDismiss, customMessage, requiredAuthDomain, title, buttonText
+}) => {
+  // State
```

Comment:
is this comment needed?

Reply:
oh I see these are throughout, pointing out the code area's function nvm I think that's fine

Reply:
We (@Shakespeared ) found these actually very nice and also appreciated the consistent ordering of the code in your changes (i.e` //State ` `//Helpers` `//Lifecycle` `//Render`) made the readability much nicer thanks!